src/Handler/CurlFactory.php

```diff
@@ -147,7 +147,7 @@ private static function finishError(
 
         // Retry when nothing is present or when curl failed to rewind.
         if (empty($easy->options['_err_message'])
-            && (!$easy->errno || $easy->errno == 65)
+            && (!$easy->errno || $easy->errno == /* CURLE_SEND_FAIL_REWIND */ 65)
```

Comment:
When is this constant not set? Was added at least 10 years ago, wasn't it?

Reply:
I don't think so. See php/php-src#4079.

Reply:
Oh, I see. They never made it into PHP-land. ;)